### PR TITLE
Fixes debian-8 bootstrap plus add specific platform dependent loading scripts

### PR DESCRIPTION
**Pull request by purbon**
Hi,
 this PR provides a fix for #5338 by providing specific bootstrap script for debian-8, it also include small change necessary to make sure all packages use the new schema name (see #5100 for details).

This PR also adds:

* custom script for `sles-12` to make sure it gets the latest java8, including also a general suse bootstrap script to download latest rpm for it.
* custom script for `ubuntu-1404` to make sure it updates the CA certificates, basically providing the solution need for #5337

_Important:_

This way this PR solve the specific bootstrap scripts is different than the one proposed in #5337, before moving forward with both we should make sure to get a common solution for it.

### CODE REVIEW

File: qa/platform_config.rb [hunk 1]
```diff
@@ -1,9 +1,34 @@
...
+
+    def initialize_bootstrap_scripts(data)
+      @bootstrap = OpenStruct.new(:privileged     => "sys/#{type}/bootstrap.sh",
+                                  :non_privileged => "sys/#{type}/user_bootstrap.sh")
+      ##
+      # for now the only specific boostrap scripts are ones need
+      # with privileged access level, whenever others are also
+      # required we can update this section as well with the same pattern.
+      ##
+      @bootstrap.privileged = "sys/#{type}/#{name}/bootstrap.sh" if data["specific"]
```
Comment (ph) [inline qa/platform_config.rb]: OK with this strategy to override the default scripts

File: qa/platform_config.rb [hunk 2]
```diff
@@ -1,9 +1,34 @@
...
+    def initialize(name, data)
+      @name = name
+      @box  = data["box"]
+      @type = data["type"]
+      initialize_bootstrap_scripts(data)
+    end
+
+    private
+
+    def initialize_bootstrap_scripts(data)
```
Comment (ph) [inline qa/platform_config.rb]: Can we come up with a better name? Its more like a configure to me.

Comment (purbon) [inline qa/platform_config.rb]: what do you prefer? something like configure_bootstrap_scripts?

**Comment by ph** [comment]
Tested with
```
vagrant up debian-8
vagrant up ubuntu-1404
vagrant up sles-12
```
minor comment, LGTM

**Pull request closed without merging**